install-dev/upgrade/php/ps_1760_update_configuration.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+/**
+ * Init new configuration values
+ */
+function ps_1760_update_configuration()
+{
+    Configuration::updateValue('PS_MAIL_THEME', 'modern');
```

Comment:
Are we really want to force the theme?

Reply:
Well, for fresh installs we do. For upgrade that's something that needs to be discussed and defined with @colinegin and Cecile But it can be fine tuned during freeze
It's just a matter of "what default values do we set?"

Besides for now the generator doesn't overwrite templates if they already exist to avoid erasing modifications from the user. And there's a setting page which was added to select your email theme if you want to keep the classic one.